XavierC4Q/Mood-Tracker-NextJS
# google auth added

## Overview
Enable users to use their Google account to sign in.

## Notes
You need to `npm i` before starting
You will need a `.env.development` file at the root of your project with the Google creds (ask Xavier)

## Test Plan
* Go to `api/auth/signin` and verify you see a button to sign in using your gmail.
* Verify that you can select your gmail account and get redirected back to the home page.
* Open your Network tab in the Chrome dev tools. Find the request to `http://localhost:3000/api/auth/session` and verify that the response includes your user info.
